test.js

```diff
@@ -8,7 +7,7 @@ const delay = require('delay')
 const nextIdleCallback = () => {
 	const deferred = makeDeferred()
 
-	idleish(deferred.resolve, 2000)
+	setTimeout(deferred.resolve, 2000);
```

Comment:
My thinking was roughly "when testing in the browser, I do want to assert that events happened before the next idle callback".  Arguably not worth the extra test complexity though.

If it's not worth it to keep the idle-callback-based assertion, we could just drop `nextIdleCallback` and replace it in the code with `await delay(2000)`